swagger.json

```diff
@@ -4430,7 +4430,31 @@
                     "format": "int64",
                     "description": "The amount of driving time in violation in this cycle (in ms).",
                     "example": 50400000
-                  }
+                  },
+                  "onDutyMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of on duty time in this cycle (in ms).",
+                    "example": 50400000
+                  },
+                  "driveMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of driving time in this cycle (in ms).",
+                    "example": 34200000
+                  },
+                  "pendingOnDutyMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "The amount of on duty time in this cycle for pending logs (in ms).",
```

Comment:
What is a pending log? Does the above OnDutyDurationMs value include pending logs?

Reply:
I can rephrase this. `pendingOnDutyMs` represents the amount of on duty time in this cycle, including any pending logs.